orchagent/macsecpost.h

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) 2025 Arista Networks, Inc.  All rights reserved.
```

Comment:
Could we remove these 2 lines both from .cpp and .h files.

Reply:
Thanks for catching this. Removed.